Jenkinsfile_k8s

```diff
@@ -6,20 +6,24 @@ parallel(
     // as per https://github.com/jenkins-infra/shared-tools/tree/main/terraform#jenkins-pipeline
     terraform(
       stagingCredentials: [
-        string(variable: 'ORACLE_OCI_CLI_USER', credentialsId: 'staging_oracle_oci_cli_user'),
-        string(variable: 'ORACLE_OCI_CLI_FINGERPRINT', credentialsId:'staging_oracle_oci_cli_fingerprint'),
+        string(variable: 'TF_VAR_region', credentialsId: 'oracle_oci_cli_region'),
+        string(variable: 'TF_VAR_tenancy_ocid', credentialsId: 'oracle_oci_cli_tenancy'),
```

Comment:
Don't forget to add this secret in infra.ci ;)

Reply:
already in with another name